ntbs-service/Pages/Notifications/Edit/Children/_TreatmentEvent.cshtml

```diff
@@ -0,0 +1,209 @@
+                                <nhs-date-input>
+                                    <nhs-date-input-item>
+                                        <nhs-form-group nhs-form-group-type="Standard">
+                                            <label nhs-label-type="Date" asp-for="FormattedEventDate.Day">Day</label>
+                                            <input v-on:blur="validate"
+                                                   is-error-input="@hasEventDateError"
+                                                   ref="dayInput"
+                                                   nhs-input-type="Date"
+                                                   fixed-width="Two"
+                                                   asp-for="FormattedEventDate.Day" />
+                                        </nhs-form-group>
+                                    </nhs-date-input-item>
+                                    <nhs-date-input-item>
+                                        <nhs-form-group nhs-form-group-type="Standard">
+                                            <label nhs-label-type="Date" asp-for="FormattedEventDate.Month">Month</label>
+                                            <input v-on:blur="validate"
+                                                   is-error-input="@hasEventDateError"
+                                                   ref="monthInput"
+                                                   nhs-input-type="Date"
+                                                   fixed-width="Two"
+                                                   asp-for="FormattedEventDate.Month" />
+                                        </nhs-form-group>
+                                    </nhs-date-input-item>
+                                    <nhs-date-input-item>
+                                        <nhs-form-group nhs-form-group-type="Standard">
+                                            <label nhs-label-type="Date" asp-for="FormattedEventDate.Year">Year</label>
+                                            <input v-on:blur="validate"
+                                                   is-error-input="@hasEventDateError"
+                                                   ref="yearInput"
+                                                   nhs-input-type="Date"
+                                                   fixed-width="Four"
+                                                   asp-for="FormattedEventDate.Year" />
+                                        </nhs-form-group>
+                                    </nhs-date-input-item>
+                                </nhs-date-input>
+                            </nhs-fieldset>
+                        </nhs-form-group>
+                    </validate-date>
+                </nhs-grid-column>
+
+                @{
+                    var conditionalString = $"function(value) {{ return value && \"{(int)TreatmentEventType.TreatmentOutcome}\" === value }}";
```

Comment:
It might look cleaner with alternating `'` and `"` rather than escaped `\"`:
`var conditionalString = $"function(value) {{ return value && '{(int)TreatmentEventType.TreatmentOutcome}' === value }}";`?